### DISCUSSION TITLE
[SOLVED] Empty items (feeds, categories) disappear from the Feed List

### DISCUSSION THREAD

**Discussion by bambury** [martinrotter/rssguard#522] [Q&A] [answered]
Hi, I am a new user of RSS Guard and I'm a bit puzzled by the UX behavior.
Simply put, feeds where all articles are marked as read disappear from the feed list in the left pane. Whole categories do the same when all articles in all their feeds are marked as read. Empty categories are not shown and new feeds are also not shown until their articles are fetched. In contrast, meta-categories like Labels and Important Articles are visible even though they're empty.

I took a look at the documentation and in the screenshots (for example this one https://github.com/martinrotter/rssguard/blob/master/resources/docs/images/gui-dark2.png ) empty feeds are clearly visible, so I assumed this to be an option, but I haven't been able to find it. Can anyone help me? Thanks a lot in advance!

**Comment by martinrotter** [comment, marked as answer]
Hi.

Yes, RSS Guard has switchable feature "Show unread feeds only", perhaps that is the reason why you do not see some feeds in list.

![image](https://user-images.githubusercontent.com/1255302/137849724-82ac1a26-75d3-4555-889e-d2a440471efa.png)

Try to switch the feature and let me know. It can be a bug, but we will see.

**Reply by bambury** [reply]
Perfect, thank you! Exactly what I was looking for.
